docs/pages/application-access/guides/aws-console.mdx

```diff
@@ -258,7 +259,10 @@ username which you can search for to get the events history:
 
 ## Step 8. Using AWS CLI
 
-First, log into the previously configured AWS console app:
+Before beginning this step, make sure that the `aws` command line interface (CLI) tool is installed in PATH. For more information, read [Installing or updating the latest version of the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
+- `aws` Command Line Interface (CLI) tool in PATH. How to [Install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
```

Comment:
```suggestion
```
Forgot to mention that this should be removed since we are turning this into a sentence at the beginning of the section. Sorry, @stevenGravy!

Reply:
np, done @ptgott